pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "django-postgresql-reconnect"
 homepage = "https://connect.cloud.im"
```

Comment:
I will point to https://github.com/cloudblue/django-postgresql-reconnect

Reply:
done